api/v1beta1/module_types.go

```diff
@@ -163,6 +163,14 @@ type ModprobeSpec struct {
 	// The firmware(s) will be copied to the host for the kernel to find them.
 	// +optional
 	FirmwarePath string `json:"firmwarePath,omitempty"`
+
+	// ModulesLoadingOrder defines the dependency between modules loading, in case
+	// it was not created by depmod (independent modules). Format is:
+	// moduleA => moduleB => moduleC
+	// meaning: loading moduleA will require loading moduleB, and loading moduleB will require loadig moduleC.
+	// In order to load all 3 modules, moduleA shoud be defined in the ModuleName parameter of this struct
+	// +optional
+	ModulesLoadingOrder string `json:"modulesLoadingOrder,omitempty"`
```

Comment:
Could this be a list instead? I do not see the point of this special syntax.

Reply:
I think that special syntax make its more clear regarding the ordering.

Reply:
fixed